package.json

```diff
@@ -2153,6 +2159,7 @@
         "sleep-promise": "^9.1.0",
         "strip-ansi": "^5.2.0",
         "tcp-port-used": "^1.0.1",
+        "tinycolor2": "^1.4.2",
```

Comment:
Where is this used?

Reply:
To be removed! I was using it to determine if a background was light/dark, but vscode does that automatically, so we just set colours based on the vscode style. Meant to remove, my bad.